bip-musig2.mediawiki

```diff
@@ -135,13 +138,17 @@ However, if ''PartialSigVerify'' succeeds for all partial signatures then ''Part
 
 ==== Tweaking ====
 
+The aggregate public key can be ''tweaked'', which modifies the key as defined in the [[#tweaking-definition|Tweaking Definition]] subsection.
+In order to apply a tweak, the KeyGen Context output by ''KeyAgg'' is provided to the ''ApplyTweak'' algorithm with the ''is_xonly_t'' argument set to false for ordinary tweaking and true for X-only tweaking.
+The resulting KeyGen Context can be used to apply another tweak with ''ApplyTweak'' or obtain the aggregate public key with ''GetPubkey''.
+
 In addition to public keys, the ''KeyAgg'' algorithm accepts tweaks, which modify the aggregate public key as defined in the [[#tweaking-definition|Tweaking Definition]] subsection.
 For example, if ''KeyAgg'' is run with ''v = 2'', ''is_xonly_t<sub>1</sub> = false'', ''is_xonly_t<sub>2</sub> = true'', then the aggregate key is first ordinarily tweaked with ''tweak<sub>1</sub>'' and then X-only tweaked with ''tweak<sub>2</sub>''.
 
 The purpose of specifying tweaking is to ensure compatibility with existing uses of tweaking, i.e., that the result of signing is a valid signature for the tweaked public key.
 The MuSig2 algorithms take arbitrary tweaks as input but accepting arbitrary tweaks may negatively affect the protocol's security.
 Instead, signers should obtain the tweaks according to other specifications.
-This typically involves deriving the tweaks from a hash of the aggregate public key and some other information.
+This typically involves deriving the tweaks from a hash of the aggregate public key (obtained with ''GetPubkey'' or ''GetLegacyPubkey'') and some other information.
```

Comment:
This solution isn't ideal either because the definition of a pubkey is already that it's a byte array. How about replacing `GetLegacyPubkey` with `GetPubkey33` to avoid the word "compressed" because it's not compressed compared to the (x-only) pubkey?

Reply:
Hm, I think `GetPubkey33` is better but I still find it somewhat confusing because for tweaking we different terms "ordinary" and "xonly". But I see that a "pubkey" here is by default an xonly pubkey, so that doesn't really make sense either. :/

Sorry this is not constructive. I don't have a better suggestion than we have now but if you have other ideas, it may be worth mentioning them.